### PR TITLE
Atree active box

### PR DESCRIPTION
**Pull request by hppeng-wynn**
Sliders display and work now

stat scaling works

left to do: spell specific modifiers

### CODE REVIEW

File: js/atree_constants.js [hunk 1]
```diff
@@ -1405,7 +1405,7 @@ const atrees = {
                 "type": "raw_stat",
                 "bonuses": [{
                     "type": "stat",
-                    "name": "mdCritPct",
+                    "name": "critDamPct",
```
Comment (ferricles) [inline js/atree_constants.js:1411]: is this a new ID

**Review by ferricles — APPROVED** [review]
mostly lgtm but the whole [damMult/defMult].[subpart]:[affected spell] convention needs to be changed

File: js/atree_constants.js (hunk 1 above)
Comment (hppeng-wynn) [reply js/atree_constants.js:1411]: yes

Comment (hppeng-wynn) [reply js/atree_constants.js:1412]: not implemented... yet,,,,,

**Pull request merged**